platforms/ios/src/TangramMap/TGTypes.mm

```diff
@@ -0,0 +1,11 @@
+//
+//  TGTypes.mm
+//  TangramMap
+//
+//  Created by Karim Naaji on 4/20/17.
+//  Copyright (c) 2017 Mapzen. All rights reserved.
+//
+
+#import "TGTypes.h"
+
+NSString* const TGErrorDomain = @"TGErrorDomain";
```

Comment:
In general I like to leave those kind of global in a single compilation unit. Otherwise each includer would define its own instance of this variable.